libraries/I2S/src/pio_i2s.pio

```diff
@@ -157,6 +166,18 @@ right:
 
 % c-sdk {
 
+// doesn't yet force the same PIO as the IN/OUT program
+static inline void pio_i2s_MCLK_program_init(PIO pio, uint sm, uint offset, uint MCLK_pin) {
+    pio_gpio_init(pio, MCLK_pin);
+		pio_sm_set_consecutive_pindirs(pio, sm, MCLK_pin, 1, true);
+    pio_sm_config sm_config = pio_i2s_mclk_program_get_default_config(offset);   
+    sm_config_set_set_pins(&sm_config, MCLK_pin, 1);
+    pio_sm_init(pio, sm, offset, &sm_config);
+		
+   // pio_sm_set_pins(pio, sm, 0); // clear pins
```

Comment:
For clarity could we remove and not just comment out un-needed code like this and the next line, please?